modules/architecture-machine-config-pools.adoc

```diff
@@ -24,3 +24,6 @@ Any node labeled with the `infra` role that is only running infra workloads is n
 ====
 
 The MCO applies updates for pools independently; for example, if there is an update that affects all pools, nodes from each pool update in parallel with each other. If you add a custom pool, nodes from that pool also attempt to update concurrently with the master and worker nodes.
+
+There might be situations where the configuration on a node does not fully match what the currently-applied machine config specifies. This state is called _configuration drift_. The Machine Config Daemon (MCD) regularly checks the nodes for configuration drift. If the MCD detects configuration drift, the MCO marks the node `degraded` until an admin corrects the node configuration. A degraded node is online and operational. However, it cannot be updated.
```

Comment:
* admin -> administrator
* Not critical, but I'd consider combining the last two sentences, that it's online/operational, but can't be updated

Same for several other places that use this/similar wording